src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1419,7 +1419,7 @@ BOOST_AUTO_TEST_CASE(ConcatAlongAxisTest)
 
     // Compute output of Concat<> layer.
     arma::Row<size_t> inputSize{outputWidth, outputHeight, outputChannel};
-    Concat<> module(inputSize, axis);
+    Concat<> module(inputSize, axis, false);
```

Comment:
Ahh, sorry this should be `true` not `false`.

Reply:
Yeah, that works locally. Lets get the tests green. Thanks a lot! :smiley: